airflow/Dockerfile

```diff
@@ -20,3 +51,38 @@ LABEL name="Apache Airflow" \
       description="This image is deployed by the Stackable Operator for Apache Airflow."
 
 COPY airflow/licenses /licenses
+
+RUN microdnf update \
+        && microdnf install -y \
+            shadow-utils \
+            python${PYTHON} \
+        && microdnf clean all
```

Comment:
This suggestion should have the same effect and make the image better for OpenShift certification. We have to update all images sometime in the future with this block.
```suggestion
# Update image and install python
RUN microdnf install -y yum python${PYTHON} \
  && yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical \
  && yum clean all \
  && microdnf clean all 
```

Reply:
Done.